.circleci/config.yml

```diff
@@ -10,6 +10,11 @@ jobs:
       HOMEBREW_VERBOSE: 1
       HOMEBREW_VERBOSE_USING_DOTS: 1
     steps:
+      - run:
+          name: Install minimal python for libxcb build
+          command: |
+            sudo apt-get update
+            sudo apt-get install --yes --force-yes python2.7-minimal
```

Comment:
Isn't `-y` sufficient (instead of `--yes --force-yes`)?

Reply:
I think `-y` == `--yes`, that's all?